libs/utils/source/ftrack_utils/extensions/overrides.py

```diff
@@ -11,6 +11,18 @@ def first_level_merge(root_extension_dict, override_dict):
     return root_extension_dict.update(override_dict)
 
 
+def is_python_extension(extension):
+    '''Return true if the extension is a python extension, false otherwise.'''
+    return not extension['extension_type'].endswith(
+        '_config'
+    ) and not extension['extension_type'].startswith('js_')
+
+
+def is_yaml_extension(extension):
+    '''Return true if the extension is a yaml extension, false otherwise.'''
+    return extension['extension_type'].endswith('_config')
+
+
```

Comment:
```suggestion
def is_python_extension(extension):
    '''Return true if the extension is a python extension, false otherwise.'''
    return not extension['extension_type'].endswith(
        '_config'
    ) and not extension['extension_type'].startswith('js_')


def is_yaml_extension(extension):
    '''Return true if the extension is a yaml extension, false otherwise.'''
    return extension['extension_type'].endswith('_config')


```

I don't really like this, I think in case is-python_extension it should verify that its really a python extension and same for yaml extension. Shouldn't be based on endswith _config. What I was proposing the other day is to has _js instead of js_ so in the original code we could have a list of extension_type to ignore, something like ignore_extension_types['_js', '_config'] the if not new_extension['extension_type'] ends with any of the ignore_extension_types....